src/projects/diff/highlight.html

```diff
@@ -107,24 +108,21 @@
       </div>
       <pre><code id="codeView"></code></pre>
     </div>
+
+    <hunk-ghost hunk-text="[[hunk.file]]"></hunk-ghost>
```

Comment:
Give this the id ghost so we can refer with this.$.ghost. This is faster than a queryselector